wgpu-hal/src/lib.rs

```diff
@@ -799,6 +803,7 @@ pub struct SamplerDescriptor<'a> {
 pub struct BindGroupLayoutDescriptor<'a> {
     pub label: Label<'a>,
     pub entries: &'a [wgt::BindGroupLayoutEntry],
+    pub partially_bound: bool,
```

Comment:
this should be in flags, like we have `InstanceFlags`, `PipelineLayoutFlags`, etc